gef.py

```diff
@@ -10182,7 +10176,7 @@ def invoke(self, args, from_tty):
 
         if argc == 1:
             prefix = argv[0]
-            names = list(filter(lambda x: x.startswith(prefix), __config__.keys()))
+            names = list([x for x in __config__.keys() if x.startswith(prefix)])
```

Comment:
I think we don't need the `list(...)` around the list comprehension.

Reply:
Good eye. The tool made some mistakes :)